packages/domains-table/src/domains-table/domains-table-ssl-cell.tsx

```diff
@@ -0,0 +1,60 @@
+import { DomainData } from '@automattic/data-stores';
+import { Icon } from '@wordpress/components';
+import { lock } from '@wordpress/icons';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import type { MouseEvent } from 'react';
+
+interface DomainsTableSSLCellProps {
+	domainManagementLink: string;
+	sslStatus: DomainData[ 'ssl_status' ];
+}
+
+export default function DomainsTableSSLCell( {
+	domainManagementLink,
+	sslStatus,
+}: DomainsTableSSLCellProps ) {
+	const translate = useTranslate();
+
+	const getSSLStatusText = () => {
+		if ( sslStatus === 'active' ) {
+			return translate( 'Active' );
+		}
+		if ( sslStatus === 'pending' ) {
+			return translate( 'Pending' );
+		}
+		if ( sslStatus === 'disabled' ) {
+			return translate( 'Disabled' );
+		}
+	};
+
+	return (
+		<div className="domains-table-row__ssl-cell">
+			{ sslStatus && (
+				<Icon
+					className={ clsx( 'domains-table-row__ssl-icon', {
+						[ 'domains-table-row__ssl-icon__active' ]: sslStatus === 'active',
+						[ 'domains-table-row__ssl-icon__pending' ]: sslStatus === 'pending',
+						[ 'domains-table-row__ssl-icon__disabled' ]: sslStatus === 'disabled',
+					} ) }
+					icon={ lock }
```

Comment:
```suggestion
					icon={ lock }
					size={ 18 }
```
Will improve styling. 18 instead of 16 aligns with the renewal icon, and also seems to be what's used in the Figma doc.